compiler/codegen/cg-expr.cpp

```diff
@@ -5010,6 +5010,10 @@ DEFINE_PRIM(GPU_SYNC_THREADS) {
 #endif
 }
 
+DEFINE_PRIM(GPU_SET_BLOCKSIZE) {
+  // processed by gpuOptimizations
+}
```

Comment:
You don't need this if you "define" the primitive with `PRIMITIVE_R` instead of `PRIMITIVE_G`. See below.

Reply:
Fixed in 9d559a8ac3b726ecaf63d09af50ae5ddac2eeafc